asv_bench/benchmarks/groupby.py

```diff
@@ -453,9 +486,11 @@ class GroupByMethods:
         ],
         ["direct", "transformation"],
         [1, 5],
+        # ["cython", "numba"],
```

Comment:
Are the cython based methods still benchmarked?

Reply:
Ack, forgot to uncomment this after debugging. Sorry!